cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -147,16 +175,16 @@ def import_handler(request, course_key_string):
                             "size": size,
                             "deleteUrl": "",
                             "deleteType": "",
-                            "url": reverse_course_url('import_handler', course_key),
+                            "url": reverse_course_url('import_handler', courselike_key),
                             "thumbnailUrl": ""
                         }]
                     })
             # Send errors to client with stage at which error occurred.
-            except Exception as exception:   # pylint: disable=broad-except
-                _save_request_status(request, key, -1)
+            except Exception as exception:  # pylint: disable=broad-except
+                _save_request_status(request, courselike_string, -1)
                 if course_dir.isdir():
                     shutil.rmtree(course_dir)
-                    log.info("Course import {0}: Temp data cleared".format(course_key))
+                    log.info("Course import %s: Temp data cleared", courselike_key)
```

Comment:
Yes, when using the logger, it handles interpolation, so you use `%`, and pass the variables in. That way a) you don't spend the time interpolating if the message isn't being logged (sometimes important in tight loops), and b) some loggers can log the message in ways that are easier to parse